.github/BOTMETA.yml

```diff
@@ -914,6 +914,7 @@ macros:
   team_openstack: emonty omgjlk juliakreger rcarrillocruz shrews thingee dagnello
   team_openswitch: Qalthos gdpak
   team_rabbitmq: chrishoffman manuel-sousa hyperized
+  team_redfish: jose-delarosa mraineri tomasg2012 billdodd
```

Comment:
Do @mraineri @tomasg2012 @billdodd know about this? Let's wait for their confirmation. Adding them might mean notification storm from bot pings.

@jose-delarosa please link to places where they've agreed to this or invite them to post that in this PR.

Reply:
Yes they know, we have an email thread. @mraineri @tomasg2012 @billdodd, can you please confirm here that you agree to be maintainers? Thank you.

Reply:
Okay, merging, then. It's better to keep such things public anyway.